BlogTemplate/Pages/Post.cshtml

```diff
@@ -17,46 +17,65 @@
 
     <h2>Comments</h2>
 
-
+    @if (User.Identity.IsAuthenticated)
+    {
+        @foreach (var comment in Model.Post.Comments)
+        {
+            string cssClass = comment.IsPublic ? "" : "deletedContent";
+            <form id="form1" method="post">
+                <div class="@cssClass">
+                    <p name="Comment-Title" class="commentTitle">@comment.AuthorName</p>
+                    <p name="Comment-Body" class="commentBody">@comment.AuthorEmail</p>
+                    <p name="Comment-Date" class="commentDate">@comment.PubDate.ToLocalTime().ToShortDateString()</p>
+                    <p name="Comment-Body" class="commentBody">@comment.Body</p>
+                    <input type="hidden" name="commentId" id="commentId" value="@comment.UniqueId" />
+                    <input type="hidden" name="slug" value="@Model.Post.Slug" />
+                    @if (comment.IsPublic)
+                    {
+                        <input type="submit" name="delete" value="Delete Comment" formaction="/ManageComment/@Model.Post.Slug?handler=DeleteComment" />

```

Comment:
Can you get this to work with the tag helper again, instead of setting the formaction manually?